Code/UsingMSBuildCopyOutputFileToFastDebug/LaunchSettingsParser.cs

```diff
@@ -1,77 +1,56 @@
 ﻿using System;
-using System.Diagnostics;
-using System.Dynamic;
-using System.Globalization;
 using System.IO;
-using System.Runtime.Serialization.Json;
-using System.Xml;
-using Lsj.Util.Collections;
-using Lsj.Util.JSON;
-using Microsoft.Build.Framework;
-//using Newtonsoft.Json;
-//using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace UsingMSBuildCopyOutputFileToFastDebug
 {
-    public class LaunchSettingsParser : Microsoft.Build.Utilities.Task
+    public class LaunchSettingsParser
     {
-        /// <inheritdoc />
-        public override bool Execute()
+        public bool Execute()
         {
             var file = Path.Combine("Properties", "launchSettings.json");
             Console.WriteLine("开始从 launchSettings 文件读取输出文件夹");
             Console.WriteLine($"开始读取{file}文件");
             if (!File.Exists(file))
             {
                 Console.WriteLine($"找不到{file}文件，读取结束");
-                return true;
+                return false;
             }
 
             var text = File.ReadAllText(file);
 
-            var o = JSONParser.Parse(text);
-            var profiles = o.profiles;
-            var data = profiles.data;
-            if (data is SafeDictionary<string, object> d)
+            var root = (JObject) JsonConvert.DeserializeObject(text);

```

Comment:
You can define your data model as below, then the deserialization works without any custom strings.

```csharp
[DataContract]
internal class LaunchSettings
{
    [DataMember(Name = "profiles")]
    public IDictionary<string, LaunchProfile> Profiles { get; set; }
}

[DataContract]
public class LaunchProfile
{
    [DataMember(Name = "commandName")]
    public string CommandName { get; set; }

    [DataMember(Name = "commandLineArgs")]
    public string CommandLineArgs { get; set; }

    [DataMember(Name = "nativeDebugging")]
    public bool NativeDebugging { get; set; }
}
```